### PR TITLE
Fix mkdir so complete path is always created + show success message on shared part import

### PR DESCRIPTION
**Pull request by Woetfin**
## Description

1. The create/import was throwing an error if the parent reconciliation_texts or shared_parts folder didn't exist yet. 

2. Import a shared part didn't give any success messages

## Type of change

- [x] Bug fix
- ~~New feature~~
- ~~Breaking change~~

## Checklist

- ~~README updated (if needed)~~
- [x] Version updated (if needed)
- ~~Documentation updated (if needed)~~

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -249,6 +249,7 @@ async function fetchSharedPart(firmId, name) {
   } else {
     fetchSharedPartByName(firmId, name);
   }
+  consola.success(`Shared part "${templateConfig.name}" imported`);
```
Comment (AgustinSilverfin) [inline index.js:252]: @Woetfin why did you move this here? now is no longer consistent with `fetchReconciliation` for example.

File: lib/templates/sharedPart.js [hunk 2]
```diff
@@ -97,11 +97,7 @@ class SharedPart {
 
   static async #findTemplateHandle(firmId, template) {
     // Search in repository
-    let handle = await fsUtils.findHandleByID(
-      firmId,
-      template.type,
-      template.id
-    );
+    let handle = fsUtils.findHandleByID(firmId, template.type, template.id);
```
Comment (AgustinSilverfin) [inline lib/templates/sharedPart.js:100]: @Woetfin any specific reason to remove the `await` ?

File: index.js (hunk 1 above)
Comment (WoetDev) [reply index.js:252]: For some reason the success message wasn't logging on the previous place, but it was logging here. Reduces duplicate code though so I think it's fine to leave it for now (need to go over all the functions either way for the partner updates, so we can align)

File: lib/templates/sharedPart.js (hunk 2 above)
Comment (WoetDev) [reply lib/templates/sharedPart.js:100]: VS Code let me auto-remove it because they said it's unnecessary, so I followed that advice
